mmdet/models/dense_heads/anchor_head.py

```diff
@@ -27,6 +27,13 @@ class AnchorHead(nn.Module):
             num_classes if None is given.
         loss_cls (dict): Config of classification loss.
         loss_bbox (dict): Config of localization loss.
```

Comment:
It's a bit too tedious for an arg description. We may just change it to something like
"The momentum factor (from 0-1) used for exponential moving average (EMA) loss normalizer. 
Default: 0, no EMA "